.github/CODEOWNERS

```diff
@@ -100,8 +100,8 @@ addons-cluster/tdengine/ @leon-inf @leon-inf @Y-Rookie @apecloud/kb-reviewers
 addons/tidb/ @csuzhangxc @cjc7373 @leon-inf @Y-Rookie @apecloud/kb-reviewers
 addons-cluster/tidb/ @csuzhangxc @cjc7373 @leon-inf @Y-Rookie @apecloud/kb-reviewers
 
-addons/vanilla-postgresql/ @ApeCloud @leon-inf @Y-Rookie @apecloud/kb-reviewers
```

Comment:
The ID was extracted from Chart.yaml, you can add `@kizuna-lek` as the second maintainer, to avoid editing the CODEOWNERS file. It's fine to keep the invalid ID `@ApeCloud` anyway.

https://github.com/apecloud/kubeblocks-addons/blob/36b2c9a24f2d5ed2c6d876fe0bbdd9268e4faed7/addons/vanilla-postgresql/Chart.yaml#L25-L27